frontend/src/components/blocks/BlocksList.jsx

```diff
@@ -63,6 +63,30 @@ export default class extends React.Component {
     }
   };
 
+  _regularLoadBlocks = async () => {
+    await this._getTopBlocks();
+    if (this.timer !== false) {
+      this.timer = setTimeout(this._regularLoadBlocks, 10000);
+    }
+  };
+
+  _getTopBlocks = async () => {
+    const blocks = await this._blocksApi.getLatestBlocksInfo();
+    const total = await this._blocksApi.getTotal();
```

Comment:
This way the queries will be done sequentially. Let's await for them in parallel:

```js
    const [blocks, total] = await Promise.all([this._blocksApi.getLatestBlocksInfo(), this._blocksApi.getTotal()]);
```

Reply:
got it